### ISSUE TITLE
Page scrolls up when interacting with an accessibility enabled legend

### ISSUE THREAD

**Issue by zoaked** [highcharts/highcharts#17868]
#### Expected behaviour
Page does not move

#### Actual behaviour
Page scrolls up

#### Live demo with steps to reproduce
https://jsfiddle.net/zgm2c7ad/3/show/

Click on the text box to start focus. Press "tab" twice to skip past all the text and move focus to the chart legend. Press "enter" or click on one of the legend items. Notice the page position moved up into the text and chart is no longer visible.

Notes: The same behavior also occurs when using a drill down data series instead of show/hide.  In our local test environments the scrolling issue also occurs when using the arrow keys to simply navigate between legend items, but thus far this has not been reproduced in jsfiddle.

#### Product version
Highcharts 10.0.0+ (this does not happen with v9.3.3)

#### Affected browser(s)
Firefox and Chrome

**Comment by exxetah** [comment]
Hi everyone,

Since the report was a while ago, are there any updates on the issue? 

Best regards

**Comment by kamil-musialowski** [comment]
Possibly related ticket: #18417

**Comment by bm64** [comment]
Hello @exxetah 

We'll take a look at the issue and prioritize it. Thanks for bringing this to our attention!

_Internal note:_ Could be related to: https://github.com/highcharts/highcharts/issues/18417

**Comment by hubertkozik** [comment]
This issue has been fixed in https://github.com/highcharts/highcharts/pull/19640.

If you would like to fix that issue in Highcharts v10, then you should use a plugin with a fix. All you have to do is copy-paste the IIFE function from the demo below.

**Live demo with a workaround:** https://jsfiddle.net/BlackLabel/3v0zdof1/

**Comment by StijnAa** [comment]
It appears this issue remains unresolved, or am I missing something? 

Live demo: [https://jsfiddle.net/Stijnaa/d9qt1nLo/](https://jsfiddle.net/Stijnaa/d9qt1nLo/)
